package.json

```diff
@@ -59,21 +59,18 @@
   },
   "repository": "github.com/sablier-labs/v2-core",
   "scripts": {
-    "benchmark": "bun run build:optimized && FOUNDRY_PROFILE=test-optimized forge test --mt testGas && bun run prettier:write",
+    "benchmark": "bun run build:optimized && FOUNDRY_PROFILE=benchmark forge test --mt \"testGas\" && bun run prettier:write",
     "build": "forge build",
     "build:optimized": "FOUNDRY_PROFILE=optimized forge build",
     "build:smt": "FOUNDRY_PROFILE=smt forge build",
     "clean": "rm -rf artifacts broadcast cache docs out out-optimized out-svg",
-    "gas:report": "forge test --gas-report --mp \"./test/integration/**/*.sol\" --nmt \"test(Fuzz)?_RevertWhen_\\w{1,}?\"",
```

Comment:
`gas:report` is still useful to print gas profile for ALL the functions. So I don't think we should remove it.

Reply:
it is the same problem for this - it is not accurate

Reply:
`forge-snapshot` creates a snapshot of each test’s gas usage. Thats why it was not useful but forge test with `--gas-report` can be useful for other functions which we do not cover in the benchmark such as `getSender`, `statusOf` etc. 

But I agree that we can remove it as we can always use forge native command to get the gas report during development.